plugin-name/plugin-name.php

```diff
@@ -66,10 +59,5 @@ function deactivate_plugin_name() {
  *
  * @since    1.0.0
  */
-function run_plugin_name() {
-
-	$plugin = new Plugin_Name();
-	$plugin->run();
-
-}
-run_plugin_name();
+$plugin = new Plugin_Name();
```

Comment:
Then this is not a way to go. Any suggestion how to get rid of pouting global scope in 5.2+?